CHARTER.md

```diff
@@ -0,0 +1,61 @@
+# vis.gl Charter
+
+Vis.gl aims to provide an open-source JavaScript framework for high-performance visualizations of large datasets on the web. It prioritizes strong support for geospatial data visualization, while also being flexible to cater to diverse non-geospatial visualization needs. It is designed to make advanced visualization techniques accessible to a wide range of developers by providing a user-friendly API and well-documented framework.
+
+## Section 0: Guiding Principles
+
+> We believe that powerful visualization should be accessible to every developer.
+
+We believe visualization tools should be declarative, composable, and interoperable—accessible to every developer and designed to work seamlessly within the broader ecosystem.
+
+Power shouldn’t come from complexity: layers, not low-level rendering; clear abstractions, not siloed systems. Our libraries should fit naturally alongside the tools developers already use, while staying modular and focused so each part can stand alone or be combined into a coherent whole.
```

Comment:
@felixpalmer @ibgreen @Pessimistress,

We previously had a copy/pasta under guiding principles and the CPC has requested (see https://github.com/visgl/tsc/pull/15#discussion_r2470738743) we put something that we believe instead. Any objections to this wording?